scripts/get_plier_pathway_data.R

```diff
@@ -0,0 +1,23 @@
+# Script to download preprocessed pathway data from PLIER
+# repo and save it in a format that can be accessed in Python.
+base_url <- 'https://github.com/wgmao/PLIER/blob'
+commit <- '5351171779d32194dcacedb8e0e804021ea75fad'
+output_dir <- file.path(getwd(), 'data', 'pathway_data')
+
+canonical_file <- sprintf('%s/%s/data/canonicalPathways.rda?raw=true',
+                          base_url, commit)
+save_filename = file.path(output_dir, 'canonicalPathways.rda')
```

Comment:
Why use `=` instead of `<-`? Are they interchangeable in this context since all assignments are in the same scope?

Reply:
I think you're right that they're interchangeable here, but really I just forgot to use `<-` (good catch)